tooling/image-updater/internal/updater/updater.go

```diff
@@ -60,19 +61,22 @@ func (u *Updater) UpdateImages(ctx context.Context) error {
 		return fmt.Errorf("logger not found in context: %w", err)
 	}
 
-	logger.V(1).Info("starting image updates", "totalImages", len(u.Config.Images))
+	totalImages := len(u.Config.Images)
+	logger.V(2).Info("starting image updates", "totalImages", totalImages)
 
 	imageNum := 0
 	updatedCount := 0
 	for name, imageConfig := range u.Config.Images {
 		imageNum++
-		logger.V(1).Info("processing image", "name", name, "source", imageConfig.Source.Image, "tagPattern", imageConfig.Source.TagPattern)
+		logger.V(2).Info("processing image", "name", name, "source", imageConfig.Source.Image, "tagPattern", imageConfig.Source.TagPattern)
 
 		imageInfo, err := u.fetchLatestDigest(ctx, imageConfig.Source)
 		if err != nil {
 			return fmt.Errorf("failed to fetch latest digest for %s: %w", name, err)
```

Comment:
The loop doesn't check if the context has been cancelled between iterations. If a user hits Ctrl+C, the tool may continue processing remaining images right?